test/manual/amp-sidebar-1.0.amp.html

```diff
@@ -52,6 +52,11 @@
   margin-right: auto;
 }
 
+.toolbar-compensation {
```

Comment:
add a class to the first element on the body to have a margin top for compensation